src/mainboard/protectli/vault_adl_n/Makefile.inc

```diff
@@ -0,0 +1,11 @@
+## SPDX-License-Identifier: GPL-2.0-only
+
+bootblock-y += bootblock.c
+
+romstage-y += gpio.c
+romstage-y += romstage_fsp_params.c
+
+ramstage-y += mainboard.c
+
+bootblock-y += die.c
+romstage-y += die.c
```

Comment:
```suggestion
smm-y += die.c
smm-y += ec.
```

Reply:
Not sure if we want to "die" with beeps in SMM.

Reply:
174bb97e2aa48b566a2610c73c1b4a201d512d53

Reply:
Maybe... Added to all stages, including SMM, anyways.